web/package.json

```diff
@@ -9,7 +9,7 @@
     "start:malte": "yarn run webpack-dev-server-ts --env config_name=malte --env dev_server",
     "start:malte-test-cms": "yarn run webpack-dev-server-ts --env config_name=malte-test-cms --env dev_server",
     "start:aschaffenburg": "yarn run webpack-dev-server-ts --env config_name=aschaffenburg --env dev_server",
-    "start:integreat-e2e": "yarn run webpack-dev-server-ts --env config_name=integreat-e2e",
+    "start:integreat-e2e": "yarn run webpack-dev-server-ts --env config_name=integreat-e2e --env dev_server",
```

Comment:
I added this, as the bundle analyzer plugin is quite verbose and actually not required here.

Reply:
Not sure if this is really a good decision. With e2e tests we want to imitate the real productive app as much as possible, and with settings the dev_server to true, we change quite a lot of the behavior. Maybe we can introduce a new variable for this?